app/assets/javascripts/discourse/app/pre-initializers/sniff-capabilities.js

```diff
@@ -46,6 +46,10 @@ export default {
       window.navigator.standalone ||
       document.referrer.includes("android-app://");
 
+    caps.wasLaunchedFromDiscourseHub = window.location.search.includes(
```

Comment:
Hmm... can we check `window.ReactNativeWebView` window context instead? We have the `isAppWebview` helper under utilities. 

This URL param feels a bit more fragile to me, yes the app always appends it, but it might not be as persistent as the JS injected by the RN webview.

Reply:
@pmusaraj shouldn't `isAppWebview` and `isiOSPWA` be moved under sniff-capabilities for consistency?

Reply:
We could move them, yeah.

Reply:
@xfalcox would you like to make this change or merge as is?